src/components/registration-view/registration-view.jsx

```diff
@@ -24,6 +27,7 @@ export function RegistrationView(props) {
     const [passwordErr, setPasswordErr] = useState("")
     const [emailErr, setEmailErr] = useState("")
 
+    // Validate function
     const validate = () => {
         let isReq = true
         if (!username) {
```

Comment:
at line 39 `password.length < 6` : you are checking here that your password is smaller than 6 characters, instead you probably want vice versa   `password.length > 6`